plugins/mcreator-localization/lang/texts.properties

```diff
@@ -877,7 +877,9 @@ blockly.block.entity_get_absorption=Get absorption health level of %1
 blockly.block.entity_get_armor_slot_item=Get item from armor slot %1 of %2
 blockly.block.entity_get_controlling_passenger=%2 Get passenger that is controlling %1
 blockly.block.entity_get_fly_speed=Get fly speed of %1
+blockly.block.entity_get_id=Get %1 Namespaced ID
```

Comment:
I'd change the texts to "Get namespaced ID of %1" and "Get UUID of %1"

Reply:
isn't the namespace ID the mod:item so minecraft:pig ?

Reply:
He meant _"Get %1 Namespaced ID" to "Get namespaced ID of %1" and "Get %1 Universal Unique Identifier (UUID)" to "Get UUID of %1"_.